res/values/strings.xml

```diff
@@ -1439,68 +1426,68 @@
     <string name="preferences__mmsc_password">MMSC Password</string>
     <string name="preferences__sms_delivery_reports">SMS delivery reports</string>
     <string name="preferences__request_a_delivery_report_for_each_sms_message_you_send">Request a delivery report for each SMS message you send</string>
-    <string name="preferences__automatically_delete_older_messages_once_a_conversation_exceeds_a_specified_length">Automatically delete older messages once a conversation exceeds a specified length</string>
+    <string name="preferences__automatically_delete_older_messages_once_a_conversation_exceeds_a_specified_length">Automatically delete old messages once a conversation exceeds a specified number of messages. Use this to keep the app from taking up too much storage space, or to prevent old messages from lingering on your phone.</string>
```

Comment:
"Automatically delete old messages once a conversation exceeds a specified number of messages. It prevents old messages from lingering on your phone and keeps the app from taking up too much storage space."

Reply:
I will write that as "and also keeps the app" because these are in essence two individual reasons for which you might want to enable this function. A user might not care about storage spage but just wants old messages erased for reasons of security.